src/pages/CustomerInvoicePage.js

```diff
@@ -251,6 +250,19 @@ const reducer = (state, action) => {
       }
       return { ...state, dataState: newDataState };
     }
+    case 'sortBy': {
+      const { data, isAscending } = state;
+      const { sortBy } = action;
+      const columnKeyToDataType = {
+        itemCode: 'string',
+        itemName: 'string',
+        availableQuantity: 'number',
+        totalQuantity: 'number',
+      };
+
+      const newData = newSortDataBy(data, sortBy, columnKeyToDataType[sortBy], isAscending);
+      return { ...state, data: newData, sortBy, isAscending: !isAscending };
```

Comment:
Should `isAscending` be negated here? Should also look at old implementation; If state sortBy was not this action's sortBy, then isAscending should be true. If state sortBy matches action's sortBy, this isAscending is false.

Reply:
ee4d03b77b3eb9ddf69486e6e1a195e790d649f0  woah oops, funky logic - this should be correct

Reply:
I commented "nice" on the commit. Idk why we can comment on commits yet.